msg/RoverDifferentialSetpoint.msg

```diff
@@ -0,0 +1,10 @@
+uint64 timestamp # time since system start (microseconds)
+
+float32 forward_speed_setpoint # [m/s or normalized [-1, 1]] Desired forward speed for the rover
+float32 yaw_rate_setpoint      # [rad/s or normalized [-1, 1]] Desired yaw rate for the rover
+float32 yaw_setpoint 	       # [rad] Desired yaw (heading) for the rover
+bool closed_loop_speed         # Use closed loop velocity controller
```

Comment:
It would call it "flag_use_closed_loop_speed" etc

Reply:
Removed the flags [here](https://github.com/PX4/PX4-Autopilot/pull/23629/commits/2c76eb214a78a493dc9e1aac76e988cf7a0ec2b3#diff-fdc56dd532fa90878add2ff88f33a2a203c15e43afa376b1105c7a466e684366R3-R7).